girder/cumulus/cumulus_plugin/assetstore.py

```diff
@@ -0,0 +1,11 @@
+from girder.api import access
+from girder.api.describe import Description, autoDescribeRoute
+from girder.models.assetstore import Assetstore
+
+@access.user
```

Comment:
We will need to add the appropriate `TokenScope` here (see #359), or else this endpoint will not be accessible to user api keys for which "Allow all actions on behalf of my user" was not checked (as is done in mongochemclient).

Reply:
Thank you for the heads up, I fixed it